asmcomp/interf.ml

```diff
@@ -148,6 +148,7 @@ let build_graph fundecl =
   (* Compute preferences and spill costs *)
 
   let rec prefer weight i =
+    let weight = max 1 weight in
```

Comment:
This adds some "floor" logic to the weight.  There is a similar "cap", which is enforced on each call site that bumps the weight (`* 8`).  It would make sense to be coherent between how the cap and the floor are implemented: either do both when entering `prefer`, or enforce them "manually" on each call site that changes the weight.

Reply:
Solved, by also upper capping it at 1000 in the entry.

Reply:
Now that weight is never divided by two, do we still need this "max 1" safeguard?